### PR TITLE
Spotify auth

### PR DESCRIPTION
**Pull request by kgensheimer**
you have to login through firebase first, then you do spotify auth

### CODE REVIEW

File: frontend/src/pages/UserProfile.jsx [hunk 1]
```diff
@@ -169,12 +170,12 @@ function UserProfile(props) {
                         </ul>
 
                         <div className="card-title">
-                            <a
+                            <Link
```
Comment (miranda-zou) [inline frontend/src/pages/UserProfile.jsx]: proba they work the same way. Does the original <a> tag not work for you?

Comment (kgensheimer) [reply frontend/src/pages/UserProfile.jsx]: yeah when using an 'a' tag it threw an error, im pretty sure you have to use 'Link' for internal redirects

File: frontend/src/pages/UserProfile.jsx [hunk 2]
```diff
@@ -206,13 +207,13 @@ function UserProfile(props) {
                             )}
                         </ul>
 
-                        <div className="webUrl">
```
Comment (miranda-zou) [inline frontend/src/pages/UserProfile.jsx]: Could you discard your change to UserProfile.jsx? It is deleting my previous update.

File: frontend/src/app/index.js [hunk 3]
```diff
@@ -28,57 +30,67 @@ const ChangePassword = lazy(() => import('../components/ChangePassword'))
...
-                        />
-                        <Route path="/">
-                            <Redirect to="/post" />
-                        </Route>
-                    </Switch>
-                </Suspense>
-            </Router>
+            <SpotifyProvider>
+                <Router>
+                    <NavBar />
```
Comment (miranda-zou) [inline frontend/src/app/index.js]: You are adding back NavBar, please delete it.

**Comment by miranda-zou** [comment]
look good to me

**Review by miranda-zou — COMMENTED** [review]
updated as request

**Pull request merged**